moped-editor/src/queries/tableLookups.js

```diff
@@ -33,7 +33,7 @@ export const TABLE_LOOKUPS_QUERY = gql`
         subcomponent_id
       }
     }
-    moped_tags(where: { is_deleted: { _eq: false } }) {
+    moped_tags(where: { is_deleted: { _eq: false } }, order_by: { name: asc }) {
```

Comment:
just updating the sort order here to match the Tags component on the project summary page